hs_core/tasks.py

```diff
@@ -374,6 +382,44 @@ def send_over_quota_emails():
             logger.debug('user ' + u.username + ' does not have UserQuota foreign key relation')
 
 
+@celery_app.task(ignore_result=True, base=HydroshareTask)
+def check_geoserver_registrations(resources):
+    # Check to ensure resources have updated web services registrations
+    if not resources:
+        cuttoff_time = timezone.now() - timedelta(days=1)
+        resources = BaseResource.objects.filter(updated__gte=cuttoff_time, raccess__public=True)
+
+    failed_resources = {}
+    err_msg = ""
+    try:
+        for res in resources:
+            response = update_web_services(
+                settings.HSWS_URL,
+                settings.HSWS_API_TOKEN,
+                settings.HSWS_TIMEOUT,
+                settings.HSWS_PUBLISH_URLS,
+                res.short_id
+            )
+            if not response['success']:
+                res_url = current_site_url() + res.get_absolute_url()
+                failed_resources[res_url] = response
+    except Exception as e:
+        err_msg = f"Exception while updating web services. Error: {str(e)}\n"
```

Comment:
This will terminate the process for any remaining resources in the list.  Would it be beneficial to try/except the update_web_service function call instead?

Reply:
Good call thanks, fixed that